charts/bandstand-cron-job/README.md

```diff
@@ -40,4 +40,6 @@
 | suspend                             | bool   | false                            | Suspend the cron job                                                                                                                                                                                                                                                                                |
 | volume.enabled                      | bool   |                                  | True if persistent volume is required                                                                                                                                                                                                                                                               |
 | volume.storage                      | string | `1Gi`                            | Amount of volume storage                                                                                                                                                                                                                                                                            |
+| volume.ephemeral                      | string |                             | Size of ephemeral storage, e.g. 10G mounted at /tmp standard emptyfile tmp directory added if not set.                                                                                                                                                                                                                                                                            |
```

Comment:
These options might be a bit confusing, as it isn't clear if you need volume.enabled set or not. I wonder if we should  change the PVC setup to just use `volume.persistent` and drop the `enabled` flag. No one is currently using the PVC stuff. Not sure.